pkg/controllers/follower/controller.go

```diff
@@ -458,6 +463,10 @@ func (c *Controller) reconcileFollower(
 		)
 		return worker.StatusError
 	} else if updated {
+		c.metrics.Store("leaders_total", len(desiredLeaders),
+			stats.Tag{Name: "namespace", Value: followerObject.GetNamespace()},
+			stats.Tag{Name: "name", Value: followerObject.GetName()},
```

Comment:
should use source object name.

Reply:
this tag has been changed to sourceName of object.